checkov/terraform/checks/resource/aws/MQBrokerAuditLogging.py

```diff
@@ -0,0 +1,17 @@
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class MQBrokerAuditLogging(BaseResourceValueCheck):
+    def __init__(self):
+        name = "Ensure MQ Broker Audit logging is enabled"
+        id = "CKV_AWS_198"
+        supported_resources = ['aws_mq_broker']
+        categories = [CheckCategories.LOGGING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
```

Comment:
can you additionally extend the `scan_resource_conf` method to check for the `engine_type`, something like that
```python
    def scan_resource_conf(self, conf):
        # https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/security-logging-monitoring-rabbitmq.html
        # Audit logging is not supported for RabbitMQ brokers.
        if conf.get("engine_type") == ["RabbitMQ"]:
            return CheckResult.UNKNOWN
        return super().scan_resource_conf(conf)
```
and of course add a test 😄